### DISCUSSION TITLE
Guidance on writing a Github externship proposal

### DISCUSSION THREAD

**Discussion by herkura** [github/india#116] [Students]
Hi,
I am interested in the Github externship program. I would like to get some guidance on writing the proposal and what is to be included in it. 
I have chosen leverice organisation.

**Comment by Surbhdey** [comment]
@herkura - The details are available on the organization [repo](https://github.com/leverice/leverice-docs) and the [doc](https://docs.leverice.com/) shared by Leverice for students to refer. 

Please feel to write to us at india-externships@github.com.

All the best 👍.

**Reply by herkura** [reply]
@Surbhdey  thanks alot.
